content/blog/2017-11-21-ochRe.md

```diff
@@ -0,0 +1,185 @@
+
+---
+slug: "ochRe"
+title: ochRe - Australia themed colour palettes
+authors:
+  - name: Alicia Allan
+    url: https://github.com/alicia-a
+  - name: Di Cook
+    url: https://twitter.com/visnut
+  - name: Ross Gayler
+    url: https://twitter.com/ross_gayler
+  - name: Holly Kirk
+    url: https://twitter.com/HollyKirk
+  - name: Roger Peng
+    url: https://twitter.com/rdpeng
+  - name: Elle Saber
+    url: https://github.com/ellesaber
+categories:
+  - blog
+topicid: ???
+date: 2017-11-21
+tags:
+  - community
+  - software
+  - ozunconf
+  - unconf
+  - meetings
+  - palettes
+  - datavis
+  - R
+---
+
+The second rOpenSci [OzUnConf](http://ozunconf17.ropensci.org/) was held in Melbourne Austrlalia a few weeks ago. A diverse range of [scientists, developers and general good-eggs](https://ropensci.org/blog/2017/10/31/ozunconf2017/) came together to make some R-magic happen and also learn a lot along the way. Before the conference began, a huge stack of projects were suggested on the unconf  [GitHub repo](https://github.com/ropensci/ozunconf17/issues). For six data-visualisation enthusiasts, one issue in particular caught their eye, and the `ochRe` package was born.
+
+![](/img/blog-images/2017-11-21-ochRe/AusElevationExamplePalettes.png)
+
+The [**ochRe**](https://github.com/ropenscilabs/ochRe) package contains colour palettes influenced by the Australian landscape, iconic Australian artists and images. OchRe is orginally the brain-child of [Di Cook](https://twitter.com/visnut), who was inspired by Karthik Ram's [`wesanderson`](https://github.com/karthik/wesanderson) package.  
```

Comment:
please put as [`ochRe`](https://github.com/ropenscilabs/ochRe)
then no need for bold - it will show up well